zstacklib/zstacklib/test/utils/env.py

```
@@ -2,20 +2,35 @@
+_LOCAL_MODE = os.environ.get('ZTEST_LOCAL_MODE') == '1'
+
+if _LOCAL_MODE:
+    envFile = os.path.expanduser("~/.zstack-test/env.yaml")
+else:
+    envFile = "/root/.zguest/envconfig.yaml"
+
+# Try to import real modules; fall back to no-ops for local mode
+try:
+    from zstacklib.utils import log, jsonobject
+    logger = log.get_logger(__name__)
+except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)